target/linux/bcm27xx/patches-6.1/950-0670-fbdev-Don-t-cancel-deferred-work-if-pagelist-empty.patch

```diff
@@ -24,11 +24,24 @@ Signed-off-by: Phil Elwell <phil@raspberrypi.com>
 
 --- a/drivers/video/fbdev/core/fb_defio.c
 +++ b/drivers/video/fbdev/core/fb_defio.c
-@@ -321,7 +321,8 @@ static void fb_deferred_io_lastclose(str
+@@ -132,7 +132,11 @@ int fb_deferred_io_fsync(struct file *fi
+ 		return 0;
+ 
+ 	inode_lock(inode);
+-	flush_delayed_work(&info->deferred_work);
++	/* Kill off the delayed work */
++	cancel_delayed_work_sync(&info->deferred_work);
++
++	/* Run it immediately */
++	schedule_delayed_work(&info->deferred_work, 0);
+ 	inode_unlock(inode);
+ 
+ 	return 0;
```

Comment:
This patch seems can be removed.

Reply:
Please see the latest commit which I think gets at your comment.  I will NOT marked this as resolved until I hear back from you.